src/CoreFx.Private.TestUtilities/src/System/Diagnostics/RemoteExecutorTestBase.Process.cs

```diff
@@ -49,13 +50,28 @@ private static RemoteInvokeHandle RemoteInvoke(MethodInfo method, string[] args,
             }
 
             // If we need the host (if it exists), use it, otherwise target the console app directly.
-            string testConsoleAppArgs = "\"" + a.FullName + "\" " + t.FullName + " " + method.Name + " " + string.Join(" ", args);
+            var testConsoleAppArgs = new StringBuilder();
+            testConsoleAppArgs.Append(ExtraParameter);
+            testConsoleAppArgs.Append("\"");
+            testConsoleAppArgs.Append(a.FullName);
+            testConsoleAppArgs.Append("\" ");
+            testConsoleAppArgs.Append(t.FullName);
+            testConsoleAppArgs.Append(' ');
+            testConsoleAppArgs.Append(method.Name);
+
+            // We need to surround each argument with quotes.
+            foreach (string arg in args)
+            {
+                testConsoleAppArgs.Append(" \"");
+                testConsoleAppArgs.Append(arg);
+                testConsoleAppArgs.Append('"');
+            }
```

Comment:
Are there uses of RemoteInvoke manually wrapping arguments in quotes today?  If so, can you remove those?